skit.Core/Identity/Entities/User.cs

```diff
@@ -9,4 +9,5 @@ public class User : IdentityUser<Guid>
     public Company Company { get; set; }
     
     public bool IsDeleted { get; set; }
+    
```

Comment:
Nie wiem czy chcemy tutaj settery ustawiać jako prywatne to trzeba będzie przegadać